rust/private/unpretty.bzl

```diff
@@ -0,0 +1,380 @@
+"""A module defining Rust 'unpretty' rules"""
+
+load("//rust/private:common.bzl", "rust_common")
+load(
+    "//rust/private:rustc.bzl",
+    "collect_deps",
+    "collect_inputs",
+    "construct_arguments",
+)
+load(
+    "//rust/private:utils.bzl",
+    "determine_output_hash",
+    "find_cc_toolchain",
+    "find_toolchain",
+)
+
+_UNPRETTY_MODES = [
```

Comment:
Can you link to the source of this list?

Reply:
I struggled to find documentation but added a command where I got them from.